src/SamplesApp/UITests.Shared/UITests.Shared.projitems

```diff
@@ -4233,6 +4240,9 @@
     <Compile Update="C:\src\Uno2\src\SamplesApp\UITests.Shared\Windows_UI_Xaml_Controls\Button\Button_Events.xaml.cs">
       <DependentUpon>Button_Events.xaml</DependentUpon>
     </Compile>
+    <Compile Update="C:\Users\pdeco\Documents\dev\Uno\src\SamplesApp\UITests.Shared\Windows_UI_Xaml_Input\Keyboard\Keyboard_Clickthrough_Modal.xaml.cs">
```

Comment:
Known VSWIN defect with shared projects causes this. It's been reported to Microsoft and until then keep an eye out for this.

```suggestion
    <Compile Update=""$(MSBuildThisFileDirectory)Windows_UI_Xaml_Input\Keyboard\Keyboard_Clickthrough_Modal.xaml.cs">
```